frontend/src/components/common/Game/Dialog/SearchCoverRom.vue

```diff
@@ -0,0 +1,240 @@
+<script setup lang="ts">
+import type { SearchCoverSchema } from "@/__generated__";
+import RDialog from "@/components/common/RDialog.vue";
+import romApi from "@/services/api/rom";
+import storeRoms, { type SimpleRom } from "@/stores/roms";
+import type { Events } from "@/types/emitter";
+import type { Emitter } from "mitt";
+import { inject, onBeforeUnmount, ref } from "vue";
+import { useDisplay } from "vuetify";
+
+// Props
+const { lgAndUp } = useDisplay();
+const show = ref(false);
+const romsStore = storeRoms();
+const rom = ref<SimpleRom | null>(null);
+const searching = ref(false);
+const searchTerm = ref("");
+const games = ref<SearchCoverSchema[]>();
+const filteredGames = ref<SearchCoverSchema[]>();
+const panels = ref([0]);
+const emitter = inject<Emitter<Events>>("emitter");
+const type = ref("all");
+emitter?.on("showSearchCoverDialog", (romToSearch) => {
+  rom.value = romToSearch;
+  searchTerm.value = romToSearch.name || romToSearch.file_name_no_tags || "";
+  show.value = true;
+  searchCovers();
+});
+
+// Functions
+async function searchCovers() {
+  games.value = undefined;
+
+  if (!rom.value) return;
+
+  // Auto hide android keyboard
+  const inputElement = document.getElementById("search-text-field");
+  inputElement?.blur();
```

Comment:
auto-hide after the user wrote their search right?

Reply:
The opossite, both the ``manual match`` button and ``cover search`` button searches automatically when opened, and the text box wont work until the search is complete, so showing the mobile keyboard automatically is pointless. This is to avoid that behaviour and make the user click the text bar if want to search something different